src/lib.rs

```diff
@@ -160,7 +160,7 @@ impl Config {
         }
     }
     fn is_high(self, mask: u8) -> bool {
-        (self.bits & mask) != 0
+        (self.bits & mask) == mask
```

Comment:
Any reason for this change? Seems slightly more difficult to understand and (I'm guessing) leading to less efficient assembly.

Reply:
Mainly only for the `is_high(BF::TEMP_EN)`, which is checking two bits.

Reply:
Reverted this change again.